### PR TITLE
Move info and ping commands to miscellaneous dir

### PR DESCRIPTION
**Pull request by KevinBacabac**
## Summary of Changes
Move info and ping into miscellaneous folder. Prefix the command class with Miscellaneous.

## Related Issues
https://github.com/uwcsc/codeybot/issues/303

## Steps to Reproduce
Confirm .info and .ping work as expected.

## Demonstration of Changes
Info and ping work as intended. No change in output.

### CODE REVIEW

File: src/commands/miscellaneous/info.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ const infoCommandDetails: CodeyCommandDetails = {
   subcommandDetails: {}
 };
 
-export class InfoCommand extends CodeyCommand {
+export class MiscellaneousCommand extends CodeyCommand {
```
Comment (Picowchew) [inline src/commands/miscellaneous/info.ts:57]: This should be `MiscellaneousInfoCommand`?

**Review by Picowchew — CHANGES_REQUESTED** [review]
Just left a super small comment. :)

File: src/commands/miscellaneous/info.ts (hunk 1 above)
Comment (KevinBacabac) [reply src/commands/miscellaneous/info.ts:57]: Oops good catch thanks!

**Review by Picowchew — APPROVED** [review]
LGTM!

**Pull request merged**